imod/mf6/simulation.py

```diff
@@ -566,6 +566,16 @@ def _merge_fluxes(
                 cbc = cbc.where(partition_domain, other=np.nan)
             cbc_per_partition.append(cbc)
 
+        # Boundary conditions can be missing in certain partitions, as do their
```

Comment:
if we are merging cbc files, then merge_fluxes may not be the best name for this function, as fluxes are often per unit area. merge_budgets?